Task 3/system/Coding/src/app/States/InitializationState.c

```diff
@@ -23,22 +24,37 @@
 
 /* EXTERNAL FUNCTIONS *****************************************************************************/
 
-extern void InitializationState_enterDisplayNameAndRestartTimer1(void)
+extern void InitializationState_enterDisplayNameAndStartTimer1(void)
 {
+    /** Display team name */
```

Comment:
Inline comments use /* commenting style. Else will be generated in doxygen.

```suggestion
    /* Display team name */
```